atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockTestTools.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.keyvalue.cassandra;
+
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertThat;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+import org.apache.cassandra.thrift.Cassandra;
+import org.apache.cassandra.thrift.Column;
+import org.apache.cassandra.thrift.Compression;
+import org.apache.cassandra.thrift.ConsistencyLevel;
+import org.apache.cassandra.thrift.CqlResult;
+import org.apache.cassandra.thrift.CqlRow;
+import org.apache.thrift.TException;
+
+import com.google.common.collect.Iterables;
+import com.google.common.primitives.Longs;
+
+public final class SchemaMutationLockTestTools {
+
+    private final CassandraClientPool clientPool;
+    private final UniqueSchemaMutationLockTable lockTable;
+
+    public SchemaMutationLockTestTools(CassandraClientPool clientPool, UniqueSchemaMutationLockTable lockTable) {
+        this.clientPool = clientPool;
+        this.lockTable = lockTable;
+    }
+
+    // for handling old non-heartbeat lock values
+    public CqlResult setLocksTableValue(long lockId) throws TException {
```

Comment:
method name should reflect that it sets an old style lock. (and either make the comment a javadoc so it's more visible or remove it if the method name explains it)